src/blocks/TrustpilotBlock/TrustpilotBlock.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useRef } from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { ContentWrapper } from 'components/blockHelpers'
+import { ContextContainer } from 'components/containers/ContextContainer'
+
+const TrustpilotWrapper = styled(ContentWrapper)`
+  && {
+    max-width: 1200px;
+  }
+`
+
+export const TrustpilotBlock: React.FC = () => {
+  const trustpilotRef = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
+    const castedWindow = window as any
+    if (castedWindow && castedWindow.Trustpilot) {
+      castedWindow.Trustpilot.loadFromElement(trustpilotRef.current, true)
+    }
+  }, [])
+
+  return (
+    <TrustpilotWrapper>
+      <ContextContainer>
+        {({ currentLocale }) => (
+          <div
+            ref={trustpilotRef}
+            className="trustpilot-widget"
+            data-locale={currentLocale.label === 'se' ? 'sv-SE' : 'en-US'}
```

Comment:
Yes it's not beautiful, but it gets the job done for now :)

Reply:
Lol you knew I would comment on this 😄 
OK fine, let's do it like this for now (but I guess it's not _that_ big of a fix to add the `data-locale` as a property on the `locales` objects right?)

Reply:
That would work for sure! 😃  But wouldn't it be confusing to add `en-US` to dk and no `locales` object, or maybe not? Since this will only be used in one file.

It would be nice IMO to have a hook later on to get `currentLocale` so we can keep the component a bit cleaner and we don't we don't have to use `<ContextContainer>`

Reply:
I don't think it would be that confusing, if the property is named so it's clear it's only used for TrustPilot. And I can see what you mean with that we only use it in one file, but we also have the `adtractionSrc` property on the `locales` which are also only used in one place. I think it's nice to just have all that data in one place and make the actual components a bit cleaner without too many `if` and ternaries. Because I can see a scenario pretty soon that we'll have more reviews in Norway (in Norwegian) and then we'd have to add that case too to the conditional.

But I totally agree we should do a remake to how we use context and get rid of the `ContextContainer` by utilising the `useContext` hook for sure! Let's add that do the infinite list of todo's! 😀

Reply:
Ok, fair enough. 👍  Is this a deal breaker now or can we make a new PR on monday since I have meetings the rest of the day and we want it out today 🙏

Reply:
OK, we can merge it and I can fix it in another branch later today or on Monday. 🤖